vcpkg/concepts/features.md

```diff
@@ -0,0 +1,167 @@
+---
+title: Understanding features in vcpkg
+description: This article explains the concept of features, their capabilities, and their expected contents in detail.
+author: JavierMatosD
+ms.author: javiermat
+ms.date: 02/21/2024
+ms.topic: concept-article
+---
+
+# Concept: features
+
+## Features
+
+Features represent sets of functionality, behavior, and dependencies that can be selectively added to a package or project upon installation. 
+
+By design, features should follow these principles:
+
+- Additive: enabling feature A should provide new functionality otherwise missing from the package without disabling any other functionality.
+- Non-exclusive: enabling feature A should not prevent other features from being installed.
```

Comment:
```suggestion
- Additive: enabling a feature should provide new functionality otherwise missing from the package without disabling any other functionality.
- Non-exclusive: enabling a feature should not prevent other features from being installed.
```

After reading it again, I think indeterminate articles are appropriate here.